### PR TITLE
fix issue when saving Bert model

### PR DESCRIPTION
**Pull request by CRGONCALVES**
I have an issue when saving the Bert Model with the code below :

```
import joblib
_ = joblib.dump(nn_model,"./data/nn_model.pickle",compress=True)
```
 

 
I fix this issue by adding the code :  if "embedding_matrix"] in dict_attr:  in the file train.py (see below) :
 
```
def __getstate__(self):
        """Method called before serialization for a specific treatment to save
        model weight and structure instead of standard serialization."""
        dict_attr = dict(self.__dict__)
        if "model" in dict_attr:
            del dict_attr["model"]
            if "embedding_matrix"] in dict_attr:
                del dict_attr["embedding_matrix"]
            del dict_attr["pretrained_embedding"]
        return dict_attr
```

**Test Configuration**:
* OS: windows 10
* Python version: 3.8.8
* Melusine version: 2.3.2

### CODE REVIEW

**Comment by CRGONCALVES** [comment]
import joblib
_ = joblib.dump(CamemBert_model,"./data/CamemBert_model.pickle",compress=True)

File: melusine/models/train.py [hunk 1]
```diff
@@ -214,7 +214,8 @@ def __getstate__(self):
         dict_attr = dict(self.__dict__)
         if "model" in dict_attr:
             del dict_attr["model"]
-            del dict_attr["embedding_matrix"]
+            if "embedding_matrix" in dict_attr:
```
Comment (hugo-quantmetry) [inline melusine/models/train.py:217]: I think the condition:
`if "embedding_matrix" in dict_attr`
Could have the same indentation as :
`if "model" in dict_attr`

(The second if statement is not dependent on the first one)

File: .eggs/README.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```
Comment (hugo-quantmetry) [inline .eggs/README.txt:1]: I think an issue occurred with the git commit process,
It seems like your entire directory was pushed including undesired files

**Comment by hugo-quantmetry** [comment]
Hello Carlos,
Thank you for your contribution!
I added some comments: 
- a minor one to fix indentation in train.py 
- a larger issue concerning files specific to you (or your company) that have been pushed to the open-source package

**Comment by TFA-MAIF** [comment]
Fixed PR has been merge.

**Pull request closed without merging**